docs/.vuepress/configs/sidebar.ts

```diff
@@ -29,6 +29,12 @@ export const sidebarEn: SidebarConfig = [
       "/content/sre/cloud-provider/binero.md",
     ],
   },
+  {
+    text: "Cluster Management",
+    collapsible: true,
+     children: [
+       "/content/sre/cluster-management/imgs/hibernating-your-cluster.md",],
```

Comment:
This path is wrong, it should be `/content/sre/cluster-management/hibernating-your-cluster.md`